.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Build Debezium Incubator
```

Comment:
@ani-sha I belive this is completely unnecessary as these are steps specific for Travis only. Maven is already present in the ubuntu-latest env.

Also I belive that the `cache` shoudl be present on connector workflows as this is the place where the cache is needed.

Reply:
@jpechane Best to remove this workflow as we don't need the `webhooks` too. And adding the `cache` to connector workflows.